assets/data/effects/wilderness/encounter_wilderness_familyBusiness_revision.json

```diff
@@ -0,0 +1,13511 @@
+{
+"id": "encounter_wilderness_familyBusiness_revision",
```

Comment:
Is this the name & ID of the comic in the base game? If we want to override the original event with our text, we'll need to make sure we match :)

Reply:
Yep. This is the same ID as the original event. When I try looking at whether Family Business can trigger on a wilderness tile, the updated event is the only one to show up, so the mod does properly replace this event now.